routers/web/repo/blame.go

```diff
@@ -91,9 +93,16 @@ func RefBlame(ctx *context.Context) {
 	ctx.Data["FileSize"] = fileSize
 	ctx.Data["FileName"] = blob.Name()
 
+	var tplName templates.TplName
+	if ctx.FormBool("only_content") {
+		tplName = tplRepoHomeContent
+	} else {
+		tplName = tplRepoHome
+	}
+
```

Comment:
Why "blame" page need to show the file tree?

When click the file tree item on the blame page, it jumps to "src" page?